lib/tessel/deploy.js

```diff
@@ -100,7 +108,7 @@ actions.execRemoteCommand = function(tessel, command, filepath) {
   });
 };
 
-actions.findProject = function(file) {
+actions.findProject = function(file, single) {
```

Comment:
good call, I went with `opts` since that's what been used in the rest of the file.